plugins/nanospell/plugin.js

```diff
@@ -59,11 +59,29 @@
 		// (for example, p, li, td)
 		// and provides a mechanism for iterating over each word within,
 		// ignoring non-block elements.  (for example, span)
-		var isNotBookmark = CKEDITOR.dom.walker.bookmark(false, true);
+		var bookmarkGuardFn = CKEDITOR.dom.walker.bookmark(false, true);
```

Comment:
BTW, I don't know if you are aware, but http://docs.ckeditor.com/ can be used to search for any of the "CKEDITOR.*" concepts/functions, if you are curious about things (like what the dom walker actually is )